### ISSUE TITLE
Check if lld is used to link rust when installing from nix

### ISSUE THREAD

**Issue by Anton-4** [roc-lang/roc#677]

**Comment by skyqrose** [comment]
This seems very specific to something you were doing 4 years ago, and is unlikely to still be relevant, or picking it back up would require context that's been lost to time. Closing.

**Comment by Anton-4** [comment]
Yeah, I still know what this is about and I should still check this, going to add it to my TODOs.
